luigi/worker.py

```diff
@@ -268,6 +280,11 @@ class worker(Config):
 
     ping_interval = FloatParameter(default=1.0,
                                    config_path=dict(section='core', name='worker-ping-interval'))
+    task_ping_ratio = IntParameter(default=60,
+                                   config_path=dict(section='core', name='worker-task-ping-ratio'),
```

Comment:
Done, but that moves task-ping-ratio to be the only item in the [worker] section in configuration.rst. Can we move all the  all and keep the older names under an older version number in readthedocs?